core/block.js

```diff
@@ -704,17 +704,6 @@ Blockly.Block.prototype.getFieldValue = function(name) {
 };
 
 /**
- * Returns the language-neutral value from the field of a block.
- * @param {string} name The name of the field.
- * @return {?string} Value from the field or null if field does not exist.
- * @deprecated December 2013
```

Comment:
This function (and the function below) has been deprecated for three years. Anyone who hasn't renamed by now is probably not going to.